pkg/util/annotation.go

```diff
@@ -23,6 +23,17 @@ func MergeAnnotation(obj *unstructured.Unstructured, annotationKey string, annot
 	}
 }
 
+// ReplaceAnnotation adds annotation for the given object, replace the value if key exist.
+func ReplaceAnnotation(obj *unstructured.Unstructured, annotationKey string, annotationValue string) {
```

Comment:
Can you help add a ut to cover this method?

Reply:
done